test/expected.txt

```diff
@@ -213,9 +213,10 @@
 || 0 persons listed!
 || ===================================================
 || Enter command: || [Command entered:  find betsy]
-|| 
-|| 0 persons listed!
-|| ===================================================
+|| 	1. Betsy Choo Tags: [secretive]
```

Comment:
Good that you have updated the test file. However, I would recommend adding more different test scenarios as well.